samples/create-page.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {PagesClient, protos} = require('@google-cloud/dialogflow-cx');
+
+async function main(projectId, agentId, flowId, location, displayName) {
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'my-project';
+  // const agentId = 'my-agent';
+  // const flowId = 'my-flow';
+  // const displayName = 'my-display-name';
+  // const location = 'global';
+
+  // [START dialogflow_cx_create_page_sample]
+  async function createPage(projectId, agentId, flowId, location, displayName) {
+    const pagesClient = new PagesClient();
+    const createPageRequest =
+      new protos.google.cloud.dialogflow.cx.v3.CreatePageRequest();
+    const page = new protos.google.cloud.dialogflow.cx.v3.Page();
+
+    page.displayName = displayName;
+
+    createPageRequest.parent = `projects/${projectId}/locations/${location}/agents/${agentId}/flows/${flowId}`;
+    createPageRequest.page = page;
+
+    const response = await pagesClient.createPage(createPageRequest);
```

Comment:
Would this work equally well if you just did:

```suggestion
    const response = await pagesClient.createPage({
      parent: `projects/${projectId}/locations/${location}/agents/${agentId}/flows/${flowId}`,
      page: {}
    });
```

I don't know if you gain much from instantiating the protos, and it makes the API look daunting to users.